website/homepage/src/guides_template.html

```diff
@@ -111,6 +111,23 @@
       </div>
       {% endfor %}
       </div>
+    <p class='mb-2 text-sm text-gray-500'>
```

Comment:
@abidlabs @dawoodkhan82 why was this added again? Accident? 

<img width="1076" alt="Screen Shot 2022-02-17 at 3 55 42 PM" src="https://user-images.githubusercontent.com/9021060/154476598-45b8d68c-65da-429a-a82a-cd621608e572.png">

Will make a PR to revert but it got deployed to the website so curious how it happened

Reply:
Ah gosh this slipped through the PR review. Seems like we must have been working with an old version of the files and for some reason git overwrote `master`!